src/renderers/testing/ReactTestRenderer.js

```diff
@@ -47,6 +49,19 @@ function getRenderedHostOrTextFromComponent(component) {
 }
 
 class ReactTestComponent {
+  _currentElement: ReactElement;
+  _renderedChildren: null | Object;
+  _topLevelWrapper: null | ReactInstance;
+  mountChildren: (
+    children: Object,
+    transaction: ReactTestReconcileTransaction,
+    context: Object,
+  ) => void;
+  updateChildren: (
+    children: Object,
+    transaction: ReactTestReconcileTransaction,
+    context: Object,
+  ) => void;
```

Comment:
There isn't. In this case, since there's only one Object.assign, could you replace with `class ... extends ReactMultiChild` which flow should understand